src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -109,24 +93,12 @@ tests:
                 saveAs: NOCSRElements
               - name: "AttestationSignature"
                 saveAs: attestationSignature
-
-      # verification: "Step 5: Verify that the DUT responds with the CSRResponse command."
-    - label:
-          "Step 6.1: Read the commissioner root certificate from TH2's fabric.
-          Save RCAC as Root_CA_Certificate_TH2"
-      identity: "beta"
-      cluster: "CommissionerCommands"
-      command: "GetCommissionerRootCertificate"
-      response:
-          values:
-              - name: "RCAC"
-                saveAs: Root_CA_Certificate_TH2
-
-      # verification: ""
+      
+      # verification: "Verify TH2 generates the NOC, Root CA Certificate, and ICAC, and certificates are saved correctly."
```

Comment:
```suggestion
```

Reply:
There's no way to verify that and we don't really need a verification step here - it doesn't practically mean anything anyway, if they're wrong, the DUT will reject them and the test will fail.

Reply:
I think all these verification labels are just leftovers from when this test was fully manual, actually.